.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,38 @@
+name: Run tests
+
+on: push
+
+jobs:
+  tests:
+    name: Run Xcode Tests
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        xcode: ["14.2.0"]
```

Comment:
Should we use `latest` or `latest-stable` instead?

Reply:
`14.2.0` is their latest currently. I'd keep it explicitly specified just because in future we might not want to update to latest Xcode as fast as Github does. For example some third party library is broken on new Xcode and we're waiting for an update or we just don't have time to update issues related to new Xcode, so tests will be broken in the meanwhile because actions will run on new Xcode already.